test/cypress/integration/group-paying.spec.js

```diff
@@ -183,9 +183,9 @@ describe('Group Payments', () => {
     cy.getByDT('link-PaymentRowSent').click()
     cy.getByDT('payList').find('tbody').children().should('have.length', 2)
     cy.getByDT('payList').within(() => {
-      cy.getByDT('payRow').eq(1).find('td:nth-child(1)').should('contain', `user3-${userId}`)
-      cy.getByDT('payRow').eq(1).find('td:nth-child(2)').should('contain', '$100')
-      cy.getByDT('payRow').eq(1).find('td:nth-child(3)').should('contain', humanDateToday)
```

Comment:
[`.eq(1)` returns the second DOM element](https://docs.cypress.io/api/commands/eq) from the list of elements returned by the previous selector... I don't think changing that to `1` would select the second row. However, per [your screenshot here](https://user-images.githubusercontent.com/57976479/143189896-85efd93c-bf75-4a99-a2a7-0c3e29cbc921.png), it's the first row that you want selected.